python/lsst/sims/utils/healpyUtils.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import healpy as hp
+
+__all__ = ['hpid2RaDec', 'raDec2Hpid', 'healbin']
+
+
+def hpid2RaDec(nside, hpids):
+    """
+    Correct for healpy being silly and running dec from 0-180.
+
+    Parameters
+    ----------
+    nside : int
+        Must be a value of 2^N.
+    hpids : np.array
+        Array (or single value) of healpixel IDs.
+
+    Returns
+    -------
+    raRet : float (or np.array)
```

Comment:
Does it make sense to use the "underscored name if it is in radians; normal name if it is in degrees" convention here? i.e.

hpid2RaDec returns RA, Dec in degrees
_hpid2RaDec returns RA, Dec in radians

and likewise for the other methods in this module?

I've been trying to keep the other methods in sims_utils consistent with that convention, but am open to the argument that only crazy people want to go from RA, Dec in degrees to healpixels.